Draft.md

```diff
@@ -63,90 +66,92 @@ keywords_en:
+Mit der Forderung nach der Offenlegung des Quellcodes (Open Source) wird auch dem Gedanken der Nachnutzbarkeit entsprochen. Die Offenlegung wird erweitert um die Forderung, den Code frei nutzbar zur Verfügung zu stellen (Free Source) ([@stallman2001free](#stallman2001free)). Letzteres wird als FOSS (Free/Libre Open Source Software) bezeichnet und in Deutschland z. B. von anwendungsbezogenen Wissenschaftlern und Entwicklern aus der Geoinformatik ([FOSSGIS e.V.](https://www.fossgis.de), https://www.fossgis.de [23.10.2020]) oder auch von Research-Software-Engineers in der [de-RSE e.V.](https://de-rse.org/de/association.html) (https://de-rse.org/de/association.html [23.10.2020]; ([@anzt_environment_2020](#anzt_environment_2020)) vertreten.
 
-In den Leitlinien der DFG klingt die Öffnung von Software-Quellcode im Zusammenhang mit der Qualitätssicherung ([@deutsche_forschungsgemeinschaft_2019, 14-15](#deutsche_forschungsgemeinschaft_2019)) und der Herstellung von öffentlichem Zugang zu Forschungsergebnissen ([@deutsche_forschungsgemeinschaft_2019, 19](#deutsche_forschungsgemeinschaft_2019)) an.
+In den Leitlinien der DFG klingt die Öffnung von Software-Quellcode im Zusammenhang mit der Qualitätssicherung ([@deutsche_forschungsgemeinschaft_2019, 14–15](#deutsche_forschungsgemeinschaft_2019)) und der Herstellung von öffentlichem Zugang zu Forschungsergebnissen ([@deutsche_forschungsgemeinschaft_2019, 19](#deutsche_forschungsgemeinschaft_2019)) an.
 
 #### CARE-Prinzipien und Ethos
 
 In den Leitlinien der DFG werden in Leitlinie 2 und 10 ethische Aspekte im Forschungsprozess angesprochen, die unserer Meinung nach auch im Bereich der Forschungssoftware Beachtung finden sollten.
 
-Privilegien und Ungleichheiten in der altertumswissenschaftlichen Forschung werden unbewusst auch in der Forschungssoftware fortgeschrieben: Für archäologische Feldarbeit wird beispielsweise oft Software zur Datenaufnahme eingesetzt, die für die Mehrheit der Mitarbeiter vor Ort aufgrund einer Sprachbarriere nicht verständlich ist. Eine Folge kann sein, dass diese Teammitglieder sich nicht weiter qualifizieren und anspruchsvollere Aufgaben übernehmen können. Proprietäre Software oder teure Hardware benachteiligt wiederum Forscher, die mit weniger Finanzmitteln auskommen müssen.
+Privilegien und Ungleichheiten in der altertumswissenschaftlichen Forschung werden unbewusst auch in der Forschungssoftware fortgeschrieben: Für archäologische Feldarbeit wird beispielsweise oft Software zur Datenaufnahme eingesetzt, die für die Mehrheit der Mitarbeiter vor Ort aufgrund einer Sprachbarriere nicht verständlich ist. Eine Folge kann sein, dass diese Teammitglieder sich nicht weiter qualifizieren und so anspruchsvollere Aufgaben nicht übernehmen können. Proprietäre Software oder teure Hardware benachteiligt wiederum Forscher, die mit weniger Finanzmitteln auskommen müssen.
```

Comment:
> Das aufgeführte Beispiel der Sprachbarriere überzeugt mich allerdings wenig, dass dies auch eine Frage in Bezug auf die Rezension von Forschungssoftware ist. Der ungleiche Zugang zu häufig teurer proprietärer Software ist für mich schon mehr ein Argument.

Wir benötigen ein einleuchtendes Beispiel; Hier steckt vermutlich Kürzungspotential.

Reply:
Ja, allerdings ist dies ja auch der Abschnitt, in denen wir zunächst Kriterien "guter" Forschungssoftware vorstellen und erst anschließend kommen die Fragen, die wir für die Rezension vorschlagen. Ich schlage vor, an diesem Punkt die Anregung aus dem Review nicht weiter zu verfolgen.

Reply:
Ich halte das Beispiel für sehr relevant, da es meiner eigenen Erfahrung mit internationalen Teams in Südosteuropa entspricht. Es ist für mich lebensnäher und weniger Abstrakt als das zweite und zeigt mir, wie ich selbst Teil des Problems geworden bin. Aber vielleicht kann das zweite Beispiel gestärkt werden. Wie wäre es mit: 

> Teure, proprietäre Software benachteiligt Forscher, die mit weniger Finanzmitteln auskommen müssen. Bestehende Ungleichgewichte werden damit langfristig verstetigt, indem den Kollegen zeitgemäße Analysemethoden verwehrt bleiben, damit der Zugang zu hoch bewerteten Zeitschriften erschwert wird und die studentische Ausbildung in den entsprechenden Methoden stagniert. Ein solches Gefälle kann sich schon bilden, wenn in internationalen Teams für archäologische Feldarbeit Software zur Datenaufnahme eingesetzt wird, die für die Mehrheit der Mitarbeiter vor Ort aufgrund einer Sprachbarriere nicht verständlich ist. Teammitglieder können sich gegebenenfalls nicht weiter qualifizieren und so keine anspruchsvollere Aufgaben übernehmen.

Reply:
Ja, auch ich finde die Sprachbarrieren sollten unbedingt drinnenbleiben und habe dazu Erfahrungen in Bolivien sammeln dürfen. Kolonialismus, ick hör dir trapsen.
Vielleicht kann ein anderes (kürzeres) Beispiel an die CARE-Prinzipien anknüpfen: 

> Wird z.B. eine Forschungsdatenbank generiert, die nicht in der Landessprache arbeitet, werden Indigene Forscher*innen, die der Sprache des*r Ausgräber*in nicht mächtig sind, daran gehindert, die Daten souverän selbst weiter zu verarbeiten. Dies widerspricht den CARE-Prinzipien.

Reply:
Ist nun so:

> Privilegien und Ungleichheiten in der altertumswissenschaftlichen Forschung werden unbewusst auch in der Forschungssoftware fortgeschrieben. Teure, proprietäre Software benachteiligt Forscher, die mit weniger Finanzmitteln auskommen müssen, da ihnen so der Zugang zu zeitgemäßen Analysemethoden verwehrt bleibt. In weiterer Folge könnte der Zugang zu hoch bewerteten Zeitschriften erschwert werden und die studentische Ausbildung in den entsprechenden Methoden stagnieren, was bestehende Ungleichgewichte weiter ausprägt. Auch Sprachbarrieren tragen zu einem Gefälle bei: Wenn in internationalen Teams z.B. eine Datenbank verwendet wird, die nicht auch in der Landessprache verfügbar ist, werden indigene Forscherinnen unter Umständen an einer selbstständigen Analyse und Auswertung der Daten, sowie an einer Weiterqualifizierung gehindert.